### DISCUSSION TITLE
23.10.18 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#32] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 트랜잭션 관련 테스트 디버깅
  - 스레드 확인하여 비동기 작업하도록 변경
- 내일은 어떤 일을 하나요?
  - 비동기 작업 검증
  - 트랜잭션 반대 상황 검증

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 소셜 로그인 이후 사용자가 요청했던 originalUri로 redirect 하는 방법 고민: 1. session에서 가져오는 방법 2. requestCache 이용
    - 로그인 후 보호된 리소스에 redirect 되었을 때 여전히 접근되지 않는 에러 발생..🥲
- 내일은 어떤 일을 하나요?
    - 에러 해결
    - 코드 리뷰를 반영하여 코드 수정

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 트랜잭션 전파로 인한 롤백이 제대로 되지 않아 구글링중입니다..
  * 코드 리뷰를 후다닥 진행했습니다!
* 내일은 어떤 일을 하나요?
  * 구글링한 결과로 문제 해결!
